fairmq/plugins/config/Config.cxx

```diff
@@ -72,6 +73,7 @@ Plugin::ProgOptions ConfigPluginProgramOptions()
         ("shm-zero-segment",              po::value<bool          >()->default_value(false),             "Shared memory: zero the shared memory segment memory after initialization (opened or created).")
         ("shm-zero-segment-on-creation",  po::value<bool          >()->default_value(false),             "Shared memory: zero the shared memory segment memory only once when created.")
         ("shm-throw-bad-alloc",           po::value<bool          >()->default_value(true),              "Shared memory: throw fair::mq::MessageBadAlloc if cannot allocate a message (retry if false).")
+        ("shm-metadata-msg-size",         po::value<std::size_t   >()->default_value(0),                 "Shared memory: size of the zmq metadata message (values smaller than minimum are clamped to the minimum). MAY NOT BE IMPLEMENTED ON SOME libc++ PLATFORMS!")
```

Comment:
What is the reason for the libc++ note?

Reply:
Obsolete, will remove it. Was from an earlier implementation which used the PMR std library, which is not implemented everywhere.